.gitignore

```diff
@@ -31,3 +34,6 @@ nosetests.xml
 .mr.developer.cfg
 .project
 .pydevproject
+
+# mypy
+mypy-cache
```

Comment:
Ah. Yes, I should have ended the file with a newline. Check back in a few minutes.